src/App/Fossa/Main.hs

```diff
@@ -199,6 +199,17 @@ appMain = do
           let metadata = maybe analyzeMetadata (mergeFileCmdMetadata analyzeMetadata) fileConfig
 
           doAnalyze (UploadScan apiOpts metadata)
+    --
+    Log4jCommand targetDirectory -> do
+      -- We ignore filters, include all dependencies, and disregard non-standard analysis.
+      let withAllDeps = toFlag IncludeAll True
+      let withoutUnpackedArchives = toFlag UnpackArchives False
+      let withoutExperimentalAnalyzers = ModeOptions VSIAnalysisDisabled (VSI.SkipResolution $ Set.fromList []) IATAssertionDisabled BinaryDiscoveryDisabled
+      let withJsonOutput = toFlag JsonOutput True
+      let withoutFilters = AllFilters [] (FilterCombination [] []) (FilterCombination [] [])
+      let doAnalyze destination = analyzeMain targetDirectory logSeverity destination override withoutUnpackedArchives withJsonOutput (withAllDeps) withoutExperimentalAnalyzers withoutFilters analyzePreferences
```

Comment:
- [ ] Don't use analyzeMain, have minimal version (just for log4j)

Reply:
If we could get away with only analyzing Maven and Gradle projects that would be ideal, but for an MVP this works.

Reply:
We are building this into a new, specialized entrypoint, using the real analyze can have lots of unintended side effects.  For that new entrypoint, we CAN setup to only check for maven and gradle.